infrastructure/develop/run-servers.sh

```diff
@@ -6,7 +6,7 @@ if [[ ! " prod-build dev-build " =~ " $1 " ]]; then
 fi
 
 if [[ "dev-build" = "$1" ]]; then
-    docker-compose -f ../../api/docker-compose.yml up & cd ../../frontend && yarn start
+    docker-compose -f ../../api/docker-compose.yml up & cd ../../frontend && npm run start
```

Comment:
Why did you switch this to npm?

Reply:
Whoops this wasn't supposed to be included! Good catch